### PR TITLE
Update Multi column Section with no style tags

### PR DESCRIPTION
**Pull request by anishdalvi**
Solved: #151 

No more Custom CSS in `dev__multi-column.liquid`

### CODE REVIEW

File: sections/dev__multi-column.liquid [hunk 1]
```diff
@@ -19,8 +9,9 @@
     {% for block in section.blocks %}
       <div class="multi-column__item {{ section.settings.multi-column__item }}">
         {% if block.settings.image %}
-          <div class="multi-column__item-img {{ section.settings.multi-column__item-img }}">
-            {{ block.settings.image | image_url: width: 200 | image_tag }}
+          <div class="multi-column__item-img-container {{ section.settings.multi-column__item-img-container }}">
+            {{ block.settings.image | image_url: width: 200 | image_tag: alt: 'alt_text',
+                class: section.settings.multi-column__style--item-img }}
           </div>
```
Comment (mihir-bombay-studio) [inline sections/dev__multi-column.liquid:15]: Block level styling should be put inside the blocks itself. This makes the section styles smaller and manageable. Also there seems to be a problem in this file as a div tag is not closed or used properly inside an if tag can you cehck @anishdalvi

Comment (anishdalvi) [reply sections/dev__multi-column.liquid:15]: If any tag is not closed then I wouldn't have get the desired output. Also, as you said about block level styling, I have kept the name of the block `multi-column` consistent throughout the code. I just suffixed it with the new class styling. [Reference for Nesting Elements within the same block](https://blog.openreplay.com/writing-cleaner-css-code-with-bem/) 

image


Here is the SS of my code:

image

If I am wrong then could you please explain me again as I didn't understand the above comment by you :(
Thank You!

Comment (mihir-bombay-studio) [reply sections/dev__multi-column.liquid:15]: image
I'm refering to this part. The logic for this part doesn't seem to be that great. also can you check the conflicts

Comment (anishdalvi) [reply sections/dev__multi-column.liquid:15]: Okay, I will try to change this logic!

**Comment by anishdalvi** [comment]
Changes Done. Removed the previous logic and implemented it with the Tailwind CSS and also resolved the conflicts :)

File: sections/dev__logo-carousel.liquid
Comment (mihir-bombay-studio) [inline sections/dev__logo-carousel.liquid:1]: I don't think we need any changes to this section

Comment (anishdalvi) [reply sections/dev__logo-carousel.liquid:1]: Yeah, Pardon me for that. It actually had some issue with Git Conflicts :(
Now everything is set and sync with upstream.
Kindly Check!

File: sections/dev__multi-column.liquid [hunk 2]
```diff
@@ -20,7 +10,8 @@ area<!-- dev__multi-column.liquid -->
       <div class="multi-column__item {{ section.settings.multi-column__item }}">
         {% if block.settings.image %}
           <div class="multi-column__item-img {{ section.settings.multi-column__item-img }}">
-            {{ block.settings.image | image_url: width: 200 | image_tag }}
+            {{ block.settings.image | image_url: width: 200 | image_tag: alt: 'alt_text',
```
Comment (mihir-bombay-studio) [inline sections/dev__multi-column.liquid:13]: Why do we have fixed alt text here instead we could use ...image.alt

**Comment by anishdalvi** [comment]
Image Alt Text is no more hardcoded :)
Also, used assign for assigning variables value as the section and block styles for image section, this increases code readibility

**Comment by mihir-bombay-studio** [comment]
Actually we don't need to add the extra alt attribute as image tag adds [automatically ](https://shopify.dev/docs/api/liquid/filters/image_tag#image_tag-alt)

**Comment by anishdalvi** [comment]
Since, it is written in documentation of the `image_tag` that By default, the alt attribute of the  tag is set to the media alt text. So I believe in removing schemas for alt text and remove it from attribute like this

image

Correct?

**Pull request merged**